app/src/main/java/com/randomappsinc/studentpicker/listpage/ListActivity.java

```diff
@@ -42,12 +45,17 @@ protected void onCreate(Bundle savedInstanceState) {
         String listName = getIntent().getStringExtra(MainActivity.LIST_NAME_KEY);
         setTitle(listName);
 
+        boolean startFormEditPage = getIntent().getBooleanExtra(START_ON_EDIT_PAGE,false);
+
         preferencesManager = new PreferencesManager(this);
         ListTabsAdapter listTabsAdapter = new ListTabsAdapter(
                 getSupportFragmentManager(),
                 listName,
                 listTabTitles);
         nameListPager.setAdapter(listTabsAdapter);
+        if (startFormEditPage){
```

Comment:
nit: I think we should just inline getIntent().getBooleanExtra(START_ON_EDIT_PAGE, false) within this if statement.